third_party/libusb/webport/src/libusb_js_proxy_data_model.h

```diff
@@ -135,7 +135,7 @@ struct LibusbJsGenericTransferParameters {
   // Only set for output transfers.
   optional<std::vector<uint8_t>> data_to_send;
   // Only set for input transfers.
-  optional<uint16_t> length_to_receive;
+  optional<int64_t> length_to_receive;
```

Comment:
Compiler didn't warn about the integer clamping here... Thinking about ways of preventing this in the future, there's
(a) use int64_t everywhere - I don't like it since for all input structures we'll have the same problem when converting them back into C structures;
(b) add a helper like `AssignWithBoundsCheck`, which would be a template function that takes destination point and source value;
... any other options?..

Reply:
No strong opinion on my side

Reply:
Actually, maybe a static assert would be a more readable alternative. Let me try that out (in a separate PR).